Storage/Storage/Model/OrderItemAttribute+CoreDataProperties.swift

```diff
@@ -0,0 +1,16 @@
+import Foundation
+import CoreData
+
+
+extension OrderItemAttribute {
+
+    @nonobjc public class func fetchRequest() -> NSFetchRequest<OrderItemAttribute> {
+        return NSFetchRequest<OrderItemAttribute>(entityName: "OrderItemAttribute")
+    }
+
+    @NSManaged public var metaID: Int64
+    @NSManaged public var name: String
+    @NSManaged public var value: String
+    @NSManaged public var orderItem: OrderItem?
+
+}
```

Comment:
Do you think that it worth adding the optional string field `type` that [will be introduced in the future](https://a8c.slack.com/archives/CGPNUU63E/p1603926767142700?thread_ts=1602796453.088400&cid=CGPNUU63E)?

Reply:
Hmm I was thinking we can add this when the value is available in the API (the migration doesn't require a mapping model, we can set its default to empty string) - @shiki what do you think, since you're thinking about adding `type`?

Reply:
Let's probably wait because that will have to through a code review. Maybe the Core team will not like it and suggest something different. 🙂